game/src/core/Game.java

```diff
@@ -157,7 +157,7 @@ public final class Game extends ScreenAdapter {
                 removeAllSystems();
                 activeEntityStorage =
                         levelStorageMap.computeIfAbsent(currentLevel(), k -> new HashSet<>());
-                // readd the systems so that each triggerOnAdd(entity) will be called (basically
+                // read the systems so that each triggerOnAdd(entity) will be called (basically
```

Comment:
`read`..... should be `readd` 
how ironic

Reply:
Ich habe einfach IntelliJ-Typo-Fixes gemacht, ohne zu lesen, was genau der Typo war.